paparazzi-agent/src/test/java/app/cash/paparazzi/agent/InterceptorRegistrarTest.kt

```diff
@@ -13,7 +13,7 @@ class InterceptorRegistrarTest {
       Utils::class.java,
       setOf(
         "log1" to Interceptor1::class.java,
-        "log2" to Interceptor2::class.java,
```

Comment:
are all the trailing commas considered spotless violations?  if so, can we configure Spotless to allow them?

Reply:
nvm, adding the Spotless rule will add superfluous commas in currently-missing sites.